priv/repo/migrations/20190701174220_add_buckets_to_accuracy_tables.exs

```
@@ -0,0 +1,15 @@
+defmodule PredictionAnalyzer.Repo.Migrations.AddBucketsToAccuracyTables do
+  use Ecto.Migration
+  # adding value to enum type can't be done in a transaction
+  @disable_ddl_transaction true
+
+  def up do
+    execute("alter type prediction_bin add value '6-8 min'")
+    execute("alter type prediction_bin add value '8-10 min'")
+    execute("alter type prediction_bin add value '10-12 min'")
+  end
+
+  def down do
+    # can't easily remove an enum type value
```

@losvedir do we maybe want to raise an error here, the way you might throw an `IrreversibleMigration` exception in Rails? That way if you happen to roll this migration back you're not left with the false impression that the rollback actually did anything.

---

Good idea. I'll raise an `Ecto.MigrationError` which is raised in other non-reversible scenarios.